docs/philosophy.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Philosophy
+---
+
+Jekyll offers a unique philosophy when approaching the problem of static
+site generation. This core philosophy drives development and product
+decisions. When a contributor, maintainer, or user asks herself what Jekyll
+is about, the following principles should come to mind:
+
+### 1. No Magic
+
+Jekyll is not magic. A user should be able to understand the underlying
+processes that make up the Jekyll build without much reading. It should
+behave "as you'd expect."
+
+### 2. It "Just Works"
+
+The out-of-the-box experience should be that it "just works." Run `gem
+install jekyll` and it should build any Jekyll site that it's given.
+Features like auto-regeneration and settings like the markdown renderer
+should represent sane defaults that work perfectly for the vast majority of
+cases. The burden of configuration should not be placed on the user.
```

Comment:
I think **initial configuration** is what I meant here. It's the out-of-the-box experience that I'm getting at.